### ISSUE TITLE
Strange Behaviour When RightView is toggled Open and Device rotates From Portrait To Landscape

### ISSUE THREAD

**Issue by Defkut** [John-Lluch/SWRevealViewController#404]
Hi John, thanks for this class, it's been really helpful in my app dev.

I'm running into a strange behaviour and wondering if others have experience it. I could not find an issue or a pull request tackling this. 

Issue:
-->  Launch the "RevealControllerExample", (including the RearView & RightView)
-->  Toggle open the RightView
-->  Rotate the device from Portrait To Landscape:
        a view "black square", behind the FrontView but in front of the RightView is appearing for half second. The layout is resizing later than it should.

-->  Rotate the Device from Landscape to Portrait does not display this behaviour.
-->  With the RearView toggled open, this behaviour is not display.

I'm not sure I have been clear with me description, so here is a video (unlisted) I posted on youtube. 
https://www.youtube.com/watch?v=95GP481GhZ0

My Xcode: 6.2
My iPhone (5s) : 8.2
My iPad 3rd Gen : 8.2

I did not check other configurations, so I'm not sure if this is just the new autolayout  or something from the Class itself.

Thanks for any kind of help on the matter

**Comment by Defkut** [comment]
After examining other classes (drawer, stacks, etc..) that included a right menu functionality, I had to assume this issue might be a IOS autolayout problem with the shadow of the upper view behaving strangely. 
During rotation, the shadow view does not update (frame) as fast as the view itself, kind of a "Lucky Luke" effect.

I could not test it on version earlier than ios 7.0 (Xcode 6), so not sure this is still an issue for earlier ios version.

Potential Solution:
I disabled the frontViewShadowColor ([UIColor clearColor]) which obviouly disable any shadow setting for this view.
It is not a solution, but this works perfectly for me since I'm not planning on using the shadow.

If there is no input on the matter in the next 2 weeks, I'll close the issue..
